### ISSUE TITLE
Do everything on getlantern.org domain

### ISSUE THREAD

**Issue by oxtoacart** [getlantern/lantern#4262]
We're combating fakes and phishing at the moment, and having our own alternate domains like `getiantem.org` and `lantern.io` probably just makes it harder for users to tell the real thing from the fakes.  It would be really nice just to be able to tell users that if they see anything other than `getlantern.org`, run!

**Comment by percyalpha** [comment]
yes. I think everything should be from getlantern.org. All other alternative domains should be 301 to getlantern.org

**Comment by xiam** [comment]
Should we add a FAQ entry to explain which domains are official in the meantime?

**Comment by percyalpha** [comment]
We have a notice prominently displayed on the Chinese index. So adding FAQ
won't be necessary.

On Monday, May 16, 2016, José Carlos <notifications@github.com> wrote:

> Should we add a FAQ entry to explain which domains are official in the
> meantime?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/getlantern/lantern/issues/4262#issuecomment-219452387>
>


-- 
Percy Alpha(PGP
<https://pgp.mit.edu/pks/lookup?op=vindex&search=0xF30D100F7FE124AE>)
